src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -3394,6 +3394,65 @@ enum AbsoluteDirection {
   NORTHWEST
 }
 
+"""Occupancy status of a vehicle."""
+enum OccupancyStatus {
+  """Default. There is no occupancy-data on this departure."""
+  NO_DATA_AVAILABLE
+
+  """
+  The vehicle is considered empty by most measures, and has few or no passengers onboard, but is
+  still accepting passengers. There isn't a big difference between this and MANY_SEATS_AVAILABLE
+  so it's possible to handle them as the same value, if one wants to limit the number of different
+  values.
+  """
+  EMPTY
+
+  """
+  The vehicle or carriage has a large number of seats available. The amount of free seats out of
+  the total seats available to be considered large enough to fall into this category is
+  determined at the discretion of the producer. There isn't a big difference between this and
+  EMPTY so it's possible to handle them as the same value, if one wants to limit the number of
+  different values.
+  """
+  MANY_SEATS_AVAILABLE
+
+  """
+  The vehicle or carriage has a small number of seats available. The amount of free seats out of
+  the total seats available to be considered small enough to fall into this category is
+  determined at the discretion of the producer.
+  """
+  FEW_SEATS_AVAILABLE
+
+  """The vehicle or carriage can currently accommodate only standing passengers."""
+  STANDING_ROOM_ONLY
+
+  """
+  The vehicle or carriage can currently accommodate only standing passengers and has limited
+  space for them. There isn't a big difference between this and FULL so it's possible to handle
+  them as the same value, if one wants to limit the number of different values.
+  """
+  CRUSHED_STANDING_ROOM_ONLY
+
+  """
+  The vehicle is considered full by most measures, but may still be allowing passengers to
+  board.
+  """
+  FULL
+
+  """
+  The vehicle or carriage is not accepting passengers. The vehicle or carriage usually accepts
+  passengers for boarding.
+  """
+  NOT_ACCEPTING_PASSENGERS
+
+  """
+  The vehicle or carriage is not boardable and never accepts passengers. Useful for special
+  vehicles or carriages (engine, maintenance carriage, etc…). It might not make sense to show
+  these types of trips to passengers at all.
+  """
+  NOT_BOARDABLE
```

Comment:
Did we decide that we want this enum value?

Reply:
We don't like it but it doesn't cause too much harm and it makes the API a bit more future proof in case someone needs it in the future for some reason.

Reply:
We decided that we don't want this enum value.